coral-component-card/src/scripts/Card.js

```diff
@@ -378,6 +378,7 @@ const Card = Decorator(class extends BaseComponent(HTMLElement) {
     const imagesCount = images.length;
     for (let i = 0 ; i < imagesCount ; i++) {
       const image = images[i];
+      image.setAttribute("alt", "");
```

Comment:
If alt is already set, it will set that to empty string.

Reply:
I just tested that and the alt attribute will be set to "undefined"
![Screenshot 2024-11-29 at 13 06 52 (2)](https://github.com/user-attachments/assets/06a0f0f0-9fe3-4f3b-a9f4-f1395376472e)
<img width="1512" alt="Screenshot 2024-11-29 at 13 06 52" src="https://github.com/user-attachments/assets/2d5a6a39-9495-42ee-ac9d-ad8b9542649b">